.buildkite/pipeline.ml.yml

```diff
@@ -46,27 +46,7 @@
       --test_arg=--framework=tf2
       rllib/...
 
-- label: ":brain: RLlib: Learning discr. actions TF1-static-graph (from rllib/tuned_examples/*.yaml)"
```

Comment:
As we move more and more heavy testing in, we should also sometimes remove some tests: E.g. tf1.x should no longer be tested, imo.

Reply:
do we want to double check with the bigger ML team on this?
we should have a coherent story about tf1 deprecation.

Reply:
Yes, we should do this: But note that these are only tf1.x tests, so the tf version here is really 1.x. We are NOT disabling framework="tf" (tf static graph) tests here. Sorry for the confusion:
```
framework="tf": static graph (could be for both versions: tf1.x, but also tf2.x, if tf.compat.v1 mode is used)
framework="tf2": NOT static graph AND tf2.x version
```